prisma/sql/getProvidersWeekly.sql

```diff
@@ -1,28 +1,20 @@
-with provider_retrievability_weekly as (
-    select
-        date_trunc('week', date) as week,
-        provider,
-        sum(total) as total,
-        sum(successful) as successful,
-        sum(successful::float8)/sum(total::float8) as success_rate,
-        sum(coalesce(successful, 0)) as successful_http,
-        sum(coalesce(successful, 0)::float8)/sum(total::float8) as success_rate_http
```

Comment:
old bug? should be `successful_http` instead of `successful`

Reply:
Yup, thanks for catching this.